### ISSUE TITLE
[Spotlight] All catalogue pages for digitized items

### ISSUE THREAD

**Issue by devangm** [TAMULib/exhibits#101]
**URL:** https://spotlight.library.tamu.edu/spotlight/london-collection/catalog/37b363e73e0105364a49cea354f982a7
**WCAG 2.1 AA compliant? :** No
**Violation:** 
**Issues:**  Iframes item title is read as the URL link
**Notes:** When a screenreader goes over the digitized item in question there is no alt text or description. Instead it reads the fedora url. Can a title/ARIA title be assigned that will read instead?

**Comment by qtamu** [comment]
There currently is an aria label for all of the catalogued pages. The actual label is the url. It comes from mirador and may be out of scope for the code base. 

<img width="446" height="43" alt="Image" src="https://github.com/user-attachments/assets/159848f6-547c-45b7-9879-bf327e9feb08" />

**Comment by qtamu** [comment]
This work is out of scope for this sprint. The issue should be moved to Mirador
